peps/pep-0735.rst

```diff
@@ -1411,42 +1365,6 @@ for various purposes.
 This declaration allows the project author's knowledge of the appropriate tools
 for the project to be shared with all editors of that project.
 
-Exposure of package dependencies without the package itself
```

Comment:
Should we mention this use-case in the rejected "Why not support Dependency Group Includes ..." section?

Reply:
I think so. I wondered about it after opening the PR. Given that you've had the same thought independently, I'll go ahead with it.